readme.md

```diff
@@ -79,7 +79,8 @@ erDiagram
     dateTime created_at
     dateTime updated_at
     varchar name
-    text content "or would be jsonb map of content applied to section"
+    enum type  "Value of SectionType enum"
```

Comment:
Let's declare explicitly values for enum in erd, because it's more like documentation and when you see it for the first time you have no idea what is `SectionType enum`
```suggestion
    enum type  "\"header\" | \"footer\""
```